src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int bound = 100;
```

Comment:
we use CONSTANT_CASE to name constants
```suggestion
    private static final int BOUND = 100;
```